src/platforms/php/guides/laravel/usage/index.mdx

```diff
@@ -90,7 +90,7 @@ class SentryContext
 
 <Alert level="info" title="Note">
 
-If you're using log channels to log your exceptions and are also logging exceptions to Sentry in your exception handler (as you would have configured above) exceptions might end up twice in Sentry.
+If you're using log channels to log your exceptions and are also logging exceptions to Sentry in your exception handler (as you would have configured above), exceptions might end up twice in Sentry.
```

Comment:
```suggestion
If you're using log channels to log your exceptions and are also logging exceptions to Sentry in your exception handler (as you would have configured above), exceptions might show up in Sentry twice.
```

Reply:
We should probably also change `as you would have configured above` to `as you would have configured in the getting started section)` since the `getting started` section is nog longer above this alert.